### ISSUE TITLE
Getting access token

### ISSUE THREAD

**Issue by dessnip** [playgameservices/play-games-plugin-for-unity#2295]
Hi! Can i get access token if i using PlayGamesClientConfiguration without                AddOauthScope("this_scope") or i must use                .AddOauthScope("this_scope") ? 
My code:

```
PlayGamesClientConfiguration config = new PlayGamesClientConfiguration.Builder()
                .RequestIdToken()
                .RequestServerAuthCode(false)
                .Build();
            
            PlayGamesPlatform.InitializeInstance(config);
            PlayGamesPlatform.DebugLogEnabled = true;
            PlayGamesPlatform.Activate();
            Social.localUser.Authenticate(OnProcessAuthentication);

        private void OnProcessAuthentication(bool success)
        {
            Logger.PrintLog("OnProcessAuthentication: " + success);
            if (success)
            {
                Logger.PrintLog("UserName: " + GetUserName());
                Logger.PrintLog("GetIdToken: " + PlayGamesPlatform.Instance.GetIdToken());
                Logger.PrintLog("GetServerAuthCode: " + PlayGamesPlatform.Instance.GetServerAuthCode());
            }
        }
```

GetServerAuthCode and GetIdToken are empty strings

**Comment by 2rusbekov** [comment]
Just call Social.localUser.Authenticate(OnProcessAuthentication); after few seconds after PlayGamesPlatform.Activate();

**Comment by olehkuznetsov** [comment]
Should be fixed in v0.9.54
